ci/docs/continuous_integation.md

```diff
@@ -1,5 +1,20 @@
 # Continuous integration tests
 
-The OpenShift robot is used to trigger functional tests on the code base.
+Pipelines as Code triggers the pipelineRun in [.tekton](https://github.com/openshift-pipelines/pipeline-service/tree/main/.tekton) which executes the functional tests tasks for each PR, before merging it.
+
+All the tests run on pipeline staging environment, where we spun up hypershift clusters.
+
+# Steps to setup Hypershift Configuration
+
+1. Follow [ROSA Provisioning](https://github.com/openshift-pipelines/pipeline-service/tree/main/ci/docs/rosa_cluster_provision.md) documentation for getting an AWS cluster.
+2. Intall and Configure [Hypershift](https://github.com/openshift-pipelines/pipeline-service/tree/main/ci/docs/hypershift_setup.md) on it.
+
+# setup github app
+
+- Github app setup demo
+
+# Bitwarden Secret
+
+Few secrets where we create HypershiftDeployment Resources has been stored in bitwarden like ROSA cluster kubeconfig, AWS pull secret and key pair.
```

Comment:
`Few secrets where we create HypershiftDeployment Resources` -> not clear to me, can you rephrase?